### PR TITLE
Create PM article

### LINKED ISSUE #207: Create a guide to PM (Project ??? Product ??? Program ???????)
What is that? I would like to know.

### PR DESCRIPTION
**Pull request by rohit-ganguly**
closes #207 #397 

Finally lol, an article on Product Management. I wanted to take a fundamentals approach as opposed to focusing on what the role actually does (enough of the '6 figures without coding' yapping online!!)

### CODE REVIEW

File: src/guides/career/product-management.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+## Sidebar
+
+I'd advise you to do your own research on Product Management as a career before choosing it, especially for reasons besides the fact that it's a tech job that doesn't involve constant coding (this attracts a lot of folks looking for the past of least resistance, which this job is not). You'll face adversity in this role that will make you miserable if you don't at least somewhat align with the role's definition.
+
+You should also know, Product Management is an incredibly hard role to break into as a new grad and you shouldn't count on it - most PMs transition later in their careers and start in a different role. That doesn't mean you shouldn't try to break into it as a student (you absolutely should if you're interested), but don't be discouraged if you don't land it early on. Careers are long, and we're all going to make it.
+
+## The Important Basics
+
+Product Management looks different *everywhere*. Sticking to the fundamentals will take you very far in your PM career no matter what exactly Product looks like in your organization.
```
Comment (NathanBarta) [inline src/guides/career/product-management.md:27]: "no matter what exactly Product looks like in your organization" -> "no matter what the role looks like in your organization"

Ease of reading. Would this edit make sense?

File: src/guides/career/product-management.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## The Important Basics
+
+Product Management looks different *everywhere*. Sticking to the fundamentals will take you very far in your PM career no matter what exactly Product looks like in your organization. 
+
+### The first fundamental
+The first thing to remember is that the term Product Management can be defined by the three words "who", "what", and "why". Everything else in the career is to gain a clearer understanding of these three concepts.
+
+To elaborate:
+- Who (exactly) are we building this for?
```
Comment (NathanBarta) [inline src/guides/career/product-management.md:33]: I'd suggest bolding "Who", "What", "Why". 

Idk what you mean be "(exactly)", you also used that word in the previous section, so is it important?

File: src/guides/career/product-management.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+- What (exactly) are we doing? What does success look like with this feature?
+- Why should we do this (and not something else)?
+
+In the scope of an engineering team, the job of the Product Manager is to clarify the work to be done, and have a clear reason why. You aren't the one defining how the problem gets solved at the end of the day (design and engineering-wise). You can come to the table with ideas, but designers and engineers are there to handle that portion - you need to ensure the end result is a good answer to 'who', 'what', and 'why'.
```
Comment (NathanBarta) [inline src/guides/career/product-management.md:37]: "result" -> "product" or "solution"?

File: src/guides/career/product-management.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+I'd advise you to do your own research on Product Management as a career before choosing it, especially for reasons besides the fact that it's a tech job that doesn't involve constant coding (this attracts a lot of folks looking for the past of least resistance, which this job is not). You'll face adversity in this role that will make you miserable if you don't at least somewhat align with the role's definition.
+
+You should also know, Product Management is an incredibly hard role to break into as a new grad and you shouldn't count on it - most PMs transition later in their careers and start in a different role. That doesn't mean you shouldn't try to break into it as a student (you absolutely should if you're interested), but don't be discouraged if you don't land it early on. Careers are long, and we're all going to make it.
+
+## The Important Basics
+
+Product Management looks different *everywhere*. Sticking to the fundamentals will take you very far in your PM career no matter what exactly Product looks like in your organization. 
+
+### The first fundamental
```
Comment (NathanBarta) [inline src/guides/career/product-management.md:29]: Doesn't follow capitalization scheme

File: src/guides/career/product-management.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+The first thing to remember is that the term Product Management can be defined by the three words "who", "what", and "why". Everything else in the career is to gain a clearer understanding of these three concepts.
+
+To elaborate:
+- Who (exactly) are we building this for?
+- What (exactly) are we doing? What does success look like with this feature?
+- Why should we do this (and not something else)?
+
+In the scope of an engineering team, the job of the Product Manager is to clarify the work to be done, and have a clear reason why. You aren't the one defining how the problem gets solved at the end of the day (design and engineering-wise). You can come to the table with ideas, but designers and engineers are there to handle that portion - you need to ensure the end result is a good answer to 'who', 'what', and 'why'.
+
+### The second fundamental
```
Comment (NathanBarta) [inline src/guides/career/product-management.md:39]: Doesn't follow capitalization scheme

File: src/guides/career/product-management.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+## Product Management Skills
+
+The name of the game is problem identification, prioritization, and ideation. We'll cover each individually.
+
+### Problem Identification
+
+You're a Product Manager on a software product. Let's say you're focused on Business to Consumer (B2C) products like Netflix.
+
+My personal approach is to start backwards and always ask questions. What makes your company money? For Netflix, when focusing on customer features, it's  membership fees. Why do customers pay money for Netflix? Because they find content they want to watch. How do they find content to watch? Through the algorithm that suggests categories and shows. After that, you can dig deeper into specifics. What do your users look like? Who uses the most Netflix? Where are they struggling?
```
Comment (NathanBarta) [inline src/guides/career/product-management.md:54]: Extraneous space between "it's" and "membership"

File: src/guides/career/product-management.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+The name of the game is problem identification, prioritization, and ideation. We'll cover each individually.
+
+### Problem Identification
+
+You're a Product Manager on a software product. Let's say you're focused on Business to Consumer (B2C) products like Netflix.
+
+My personal approach is to start backwards and always ask questions. What makes your company money? For Netflix, when focusing on customer features, it's  membership fees. Why do customers pay money for Netflix? Because they find content they want to watch. How do they find content to watch? Through the algorithm that suggests categories and shows. After that, you can dig deeper into specifics. What do your users look like? Who uses the most Netflix? Where are they struggling? 
+
+Understanding how your business makes money, now what are the higher ups saying strategy-wise? What can your product do to align with  overall business goals? While you represent the customers, you also have to get buy-in and aligning with strategy is a must.
```
Comment (NathanBarta) [inline src/guides/career/product-management.md:56]: Extraneous space between "with" and "overall"

File: src/guides/career/product-management.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+### Problem Prioritization
+
+Ideally, this is a pretty straightforward answer once you have a good understanding of the problems to solve. This generally can be prioritized by looking at why you're building it. If you're building something to satisfy users, build something that satisfies a more important pain point. If you're looking to make money, pick the feature that has a higher chance to increase money made from your product.
+
+You should have a clear reason for prioritizing work, and if you're feeling unsure in a particular scenario, you have more to learn about your customer, the problem, your competitors, or the business.
+
+### Ideation
+
+You'll eventually bring your prioritized ideas to your stakeholders (design, engineering, leadership, etc) and they'll have their own thoughts on the question they're responsible for, *how* do we get it done?
```
Comment (NathanBarta) [inline src/guides/career/product-management.md:75]: "thoughts on the question" seems kinda out of place

Comment (NathanBarta) [reply src/guides/career/product-management.md:75]: What's "it"?

File: src/guides/career/product-management.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+You should have a clear reason for prioritizing work, and if you're feeling unsure in a particular scenario, you have more to learn about your customer, the problem, your competitors, or the business.
+
+### Ideation
+
+You'll eventually bring your prioritized ideas to your stakeholders (design, engineering, leadership, etc) and they'll have their own thoughts on the question they're responsible for, *how* do we get it done? 
+
+You can come to the table with ideas on how it should be done from a customer's perspective. Understand their job to be done (JTBD) and pain points. The design and engineering teams will handle the rest - do *not* get attached to a solution! 
+
+The most important thing is adjusting your ideas with impact/effort. Again, you are not responsible for ideating a solution - your job is to ensure that there is value provided to customers at the end of the day. If you're getting pushback on your features and you can't defend it, you need to develop a deeper understanding of the customers/problem/business/competition OR improve your communication of your ideas.
```
Comment (NathanBarta) [inline src/guides/career/product-management.md:79]: "ideas with impact/effort"

Is "with" the right word?

File: src/guides/career/product-management.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+
+At the end of it all, you'll have to prioritize scoped features (features that have had their effort to build measured). Ideally you go for low-effort high-impact features first, and work your way backwards to medium-effort high-impact, medium-effort medium-impact, etc. This is very situational but this process will look the same everywhere.
```
Comment (NathanBarta) [inline src/guides/career/product-management.md:81]: Maybe bring up the concept of "scope" in the first paragraph of this section when you are talking about the other stakeholders.

File: src/guides/career/product-management.md [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+
+Understanding your business and your users is key. A solution that makes everyone happy exists.
+
+### Problem Prioritization
+
+Ideally, this is a pretty straightforward answer once you have a good understanding of the problems to solve. This generally can be prioritized by looking at why you're building it. If you're building something to satisfy users, build something that satisfies a more important pain point. If you're looking to make money, pick the feature that has a higher chance to increase money made from your product.
+
+You should have a clear reason for prioritizing work, and if you're feeling unsure in a particular scenario, you have more to learn about your customer, the problem, your competitors, or the business.
+
+### Ideation
```
Comment (NathanBarta) [inline src/guides/career/product-management.md:73]: Title scheme not followed. Should be "Problem Ideation".

**Review by NathanBarta — CHANGES_REQUESTED** [review]

**Comment by NathanBarta** [comment]
Looks good! Definitely fills a hole in the wiki content, so thank you for the contribution! 

I've left a few minor comments, feel free to ping me if you need clarification on any of them or if you'd like to discuss.

**Comment by rohit-ganguly** [comment]
> Looks good! Definitely fills a hole in the wiki content, so thank you for the contribution!
> 
> I've left a few minor comments, feel free to ping me if you need clarification on any of them or if you'd like to discuss.

Addressed your feedback - let me know your thoughts! Thanks :)

File: src/guides/career/product-management.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+- Digging into usage data to see where users are struggling
+- Talking to users and seeing what their issues are, or running UX research studies
+- Seeing what competitors are doing and what they're prioritizing (competitive intelligence)
+- Learning about latest developments in your field, in-person at conferences/meetups or online
+
+Understanding your business and your users is key. A solution that makes everyone happy exists.
+
+### Problem Prioritization
+
+Ideally, this is a pretty straightforward answer once you have a good understanding of the problems to solve. This generally can be prioritized by looking at why you're building it. If you're building something to satisfy users, build something that satisfies a more important pain point. If you're looking to make money, pick the feature that has a higher chance to increase money made from your product.
```
Comment (NathanBarta) [inline src/guides/career/product-management.md:72]: Paragraph confuses me. Did you mean "answered" instead of "prioritized" in sentence 2?

**Pull request merged**